src/test/java/com/mitchellbosecke/pebble/extension/core/DateFilterTest.java

```diff
@@ -0,0 +1,63 @@
+package com.mitchellbosecke.pebble.extension.core;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.mitchellbosecke.pebble.template.EvaluationContext;
+
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Vladimir Loshchin
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class DateFilterTest {
+
+	@Mock
+	private EvaluationContext ctx; 
+	
+	@Test
+	public void testRussianLocale() {
+		when(ctx.getLocale()).thenReturn(new Locale("ru", "RU"));
```

Comment:
Please don't use mockito for this test. Construct a PebbleEngine et a template like all other unit tests